kratos/utilities/geometry_tester.h

```diff
@@ -519,7 +519,7 @@ class KRATOS_API(KRATOS_CORE) GeometryTesterUtility
         GeometryType& rGeometry,
         GeometryType::CoordinatesArrayType& rGlobalCoordinates,
```

Comment:
rGlobalCoordinates should also be const no?

Reply:
In this case I follow the same definition as `VerifyShapeFunctionValues` method defined just above. Why should we pass as a `const` in `VerifyShapeFunctionsSecondDerivativesValues` but not in `VerifyShapeFunctionValues`?